client/tests/integration/test_translation.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""End to end translation test."""
```

Comment:
Does this test run manually or automatically? Maybe add a comment about how to use this integration test?

Reply:
You can run it with `make test` which I added to `CONTRIBUTING.md`: https://github.com/google/dwh-migration-tools/pull/62/files#diff-dac011c24378ac8c6e23fee303e74872b2531f3867a22a62fc8e90f28f9d0db2R31.

I would really like to have CI though. Not sure what the strategy is for that in this repo. GH actions? @shevek-google ?

Reply:
Continuous integration. Running the static analyses and tests when a PR is submitted.

Reply:
Somehow I didn't notice it's already set up internally haha.